neqo-transport/src/send_stream.rs

```diff
@@ -696,6 +697,67 @@ impl SendStream {
     }
 }
 
+#[derive(Debug, Default)]
+pub(crate) struct SendStreams(BTreeMap<StreamId, SendStream>);
```

Comment:
Is there a good reason to use `.0` here?

And `pub(crate)` throughout doesn't seem necessary unless the type is going to be public.  I would just make this full public.

Reply:
I think I need to use `.0`? Or is there a better syntax I've blanked on?

re pub crate, I wasn't sure if this was a cleanliness thing we wanted to adopt across the code base, or not. I was thinking yes, and so wanted to start doing it, but don't feel strongly.